### ISSUE TITLE
Sort by actual alphabetical order

### ISSUE THREAD

**Issue by popey456963** [kogg/instant-logos#3732]
At the moment, one assumes you're sorting by some generic order that seems to follow the current order:

```
[A-Z] --> [a-z]
```

A better method would be:

```
[AaBbCcDd..YyZz]
```

Otherwise, at the moment your most popular (which after a while turns to alphabetical order), looks rather strange.

**Comment by saiichihashimoto** [comment]
This is true! We haven't put much emphasis on this since, after a bit of usage (which has happened by now) scrolling down to this section shouldn't happen often. However, we do find that people do actually scroll down quite far, so we should reconsider this suggestion. Thanks for the input!
